lib/plugins/packages.spec.js

```diff
@@ -237,7 +237,10 @@ describe('packages', () => {
           ],
           scripts: [
             '/dist/jquery.js'
-          ]
+          ],
+          meta: {
+            description: 'Add the jQuery JavaScript library to your prototype'
```

Comment:
This description is written as a description of what would happen if a user installs it rather than a description of what jQuery itself is.  I think we should use:

```
jQuery is a fast, small, and feature-rich JavaScript library. It makes things like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API that works across a multitude of browsers.
```

(that's taken from their homepage)